Puppetfile.tracking

```diff
@@ -17,8 +17,10 @@ mod 'simp-rsync_data',
   :ref => '6.2.1'
 
 mod 'simp-adapter',
-  :git => 'https://github.com/simp/simp-adapter',
-  :ref => '0.0.6'
+#  :git => 'https://github.com/simp/simp-adapter',
+#  :ref => '0.1.0'
+  :git => 'https://github.com/jeannegreulich/simp-adapter',
```

Comment:
Need to set this back to master branch on simp github once simp-adapter changes have been  been pushed.